<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Lukáš Tinkl &lt;ltinkl@redhat.com&gt;</author>
 <class>TeamWidget</class>
 <widget class="QWidget" name="TeamWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>436</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Interface name:</string>
     </property>
     <property name="buddy">
      <cstring>ifaceName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KLineEdit" name="ifaceName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Teamed connections:</string>
     </property>
     <property name="buddy">
      <cstring>teams</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="4">
      <widget class="QListWidget" name="teams">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add…</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit…</string>
       </property>
       <property name="icon">
        <iconset theme="configure">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset theme="edit-delete">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>JSON config:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="config"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="btnImport">
     <property name="text">
      <string>Import configuration from file…</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ifaceName</tabstop>
  <tabstop>teams</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnDelete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
